packages/runtime/container-runtime/src/summary/summarizer.ts

```diff
@@ -124,12 +125,20 @@ export class Summarizer extends TypedEventEmitter<ISummarizerEvents> implements
 		};
 
 		const resolvedContainer = await loader.resolve(request);
```

Comment:
Can you confirm that all codepaths here have test coverage?  I'm imagining nearly every e2e test triggers these codepaths (and we should have the compat coverage), but sometimes I'm surprised.

Reply:
All the code paths are covered indirectly by a ton of different tests